### PR TITLE
Add Prettier and Apply it

### PR DESCRIPTION
**Pull request by igprad**
If there's any file that not want to be listed, or any concern please comment it.
Added the script to `package.json`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,11 +12,14 @@
...
   "devDependencies": {
     "@types/node": "^18.14.1",
+    "prettier": "^3.3.3",
     "typescript": "^4.9.5"
   },
   "scripts": {
     "build": "tsc",
-    "start": "bun ./dist/index.js"
+    "start": "bun ./dist/index.js",
+    "check": "npx prettier . --check",
```
Comment (heronimus) [inline package.json]: @igprad minor fix: use `bunx` instead of `npx` so it can be run inside build (bun) container 🫡

Comment (igprad) [reply package.json]: Actually, I tried to used `bun prettier . --write` on my local machine, but somehow doesn't work. May you try this in your local container just to make sure that using bun also works?

Comment (igprad) [reply package.json]: I see, i missed that you mention `bunx` instead of `bun`.
Tried on my local, it working, will adjust this.

Comment (igprad) [reply package.json]: updated

**Comment by heronimus** [comment]
LGTM ✅

**Review by heronimus — APPROVED** [review]

**Pull request merged**